apidef/oas/linter_test.go

```diff
@@ -91,6 +91,8 @@ func TestXTykGateway_Lint(t *testing.T) {
 			OAuth:     nil,
 		}
 		settings.Middleware.Global.TrafficLogs.RetentionPeriod.Value = ReadableDuration(time.Minute * 10)
+		settings.Server.Protocol = "http"
+		settings.Server.Port = 3000
```

Comment:
I would enjoy this 2% more if it was grouped together with other settings.Server assignments.